### ISSUE TITLE
Column name specified more than once

### ISSUE THREAD

**Issue by c-line** [awslabs/aws-data-wrangler#313]
When joining two tables which share multiple column names (though are not necssarily ones to join on), an error is thrown that cannot return results where there is "column named "x" specified more than once". The query will work direct through a console though. For example, two tables A and B, have column names a, b, c, d each. But we only want to join on column a. 
SELECT A.\*, B.\*
FROM A 
    LEFT JOIN  B 
     ON A.a = B.a

and the resulting table should prefix any column names which are the same with A.b, B.b, A.c, B.c etc.

This is when querying Athena using wr.athena.read_sql_query()

Note, the error does not occur when using the older version of awswrangler==0.3.2 and the method read_sql_athena()

**Comment by igorborgest** [comment]
Thank you for reach out to us @c-line!

Yeah, actually it is a limitation in our new _default_ behaviour to extract data from Athena (`ctas_approach=True`) where we wrap your query into a CTAS behind the scenes an execute it to be able to fetch the result as Parquet Files. It is far away faster to fetch big result sizes, but unfortunately it is less flexible regarding to the result structure as you are seeing.

But don't worry, you can always execute your query with the regular approach passing `ctas_approach=False`:

```py3
wr.athena.read_sql_query("SELECT ...", database="...", ctas_approach=False)
```

I will update our documentation and [tutorial](https://github.com/awslabs/aws-data-wrangler/blob/master/tutorials/006%20-%20Amazon%20Athena.ipynb) to make this limitation explicit.

Let me know if it solved your issue.

**Comment by igorborgest** [comment]
Enhancement done in commit above. Release expected for version `1.7.0` next week.

**Comment by igorborgest** [comment]
@c-line , feel free to test our dev branch before the official release:

```
pip install git+https://github.com/awslabs/aws-data-wrangler.git@dev
```

**Comment by igorborgest** [comment]
Released in [1.7.0](https://github.com/awslabs/aws-data-wrangler/releases/tag/1.7.0)!

**Comment by jaqsro** [comment]
In my case, I need to join 2 tables and the join columns has the same name.  ctas_approach is already = False, but I still see the error:
InvalidArgumentValue: Please, define distinct names for your columns. Root error message: SYNTAX_ERROR: line 1:1: Column name 'xpto' specified more than once
